code/patrick/lab13v1.py

```diff
@@ -1,50 +1,180 @@
 class Player:
-    def _init_(self, name, token):
-       self.name = name
-       self.token = token # X or O
+    def _init_(self, token):
+       self.name = " "
+  
 
-class Game:
-    def _init_ (self, board):
-        self.board_1 = self
+    def player_name(self):
+        name = input(f"Enter player #1's name: ")
+        self.name = name
         
     
-    def _repr_():
-        pass
+        
+class Game:
+    def __init__ (self):
+        self.count = 0
+        self.game = ['0', '1', '2', '3', '4', '5', '6', '7', '8']
+        self.board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+       
+      
+    def instructions(self):
+        return 'Using the above example, please enter the corresponding number you wish to place an X: ' 
+
     
-    def image(self):
-        board_1 = [
-        ["1", "2", "3"],
-        ["4", "5", "6"],
-        ["7", "8", "9"]
-        ]
-        for i in 
-        return enumerate(board_1)
-
-        # for row in board_1: 
-        #     for collum in row:
-        #         return collum
-        #for x, y
-           
-   
-    def move(x, y, player):
-        pass
     
+    def first_image(self): 
+        return f"""
+        {self.game[0]}|{self.game[1]}|{self.game[2]} 
+        {self.game[3]}|{self.game[4]}|{self.game[5]}
+        {self.game[6]}|{self.game[7]}|{self.game[8]}
+        """
 
-    def clac_winner():
-        pass
+    def __repr__(self):
+        
+        return f"""
+        {self.board[0]}|{self.board[1]}|{self.board[2]} 
+        {self.board[3]}|{self.board[4]}|{self.board[5]}
+        {self.board[6]}|{self.board[7]}|{self.board[8]}
+        """
+    def move_1(self):
```

Comment:
with the updates to the Player class, you should be able to have just one `move` method:
```def move(self, player):
    # given the player object, you'll have access to the token, you don't need separate methods 
    print(player.name, player.token) # print statement is just to demonstrate what you can get out of the player object```
Also, looking at the instructions, it seems the move method should also take in an x and y value https://github.com/PdxCodeGuild/class_wallaby/blob/main/1%20Python/labs/13%20Tic%20Tac%20Toe.md

To keep the method pure and modular (able to be used in a variety of ways––imagine if this had a graphical user interface instead of a command line interface and the user clicked on a position instead of entering coordinates) you should ask for the x and y coordinates in the main game loop below, not inside the method.

Reply:
Deleted one of the def move() and am now calling it by the two objects created in main(). Talked to Pete and was told that using a list is fine and switching to coordinate structure is not needed at the moment. I have placed all the input() functions within main() loop, much cleaner.